rero_ils/modules/notifications/dispatcher.py

```diff
@@ -25,27 +25,34 @@
 from ..locations.api import Location
 
 
-class Dispatcher():
+class Dispatcher:
     """Dispatcher notifications class."""
 
     def dispatch_notification(self, notification=None):
         """Dispatch the notification."""
+
+        def not_yet_implemented(*args):
```

Comment:
to be remove?

Reply:
As explained, the best solution should be to used a lambda function. But it seems that a lambda function cannot be referenced dynamically. So this solution is the least worst I found.